README.md

```diff
@@ -16,6 +16,10 @@ source code if you already have it handy), and it will figure out all of that
 file's dependencies and sub-dependencies, passing an array of them to your
 callback. Or use the `Sync` API to get the results as a return value.
 
+Result array has non-enumerable `patterns` property with an array of globs;
+each of the found globs is joined with every path progeny looks for dependencies in.
```

Comment:
Thats why I did not push, thought you will have notes on grammar or content. Thanks for the invite & review.

I thought it would be nice to note that globs are joined with all paths from config, not just with paths progeny found matches in. I am all in for "simplify".